overlay/validation.proto

```diff
@@ -62,14 +65,22 @@ message ValidationJobRequest {
   ModelFile model_file = 20;
 }
 
-message ValidationJobResponse {
+message ExperimentResponse {
```

Comment:
This is what the old `ValidationJobResponse` was, just renamed so we can still use in experiments/benchmarks.